generation/package.json

```diff
@@ -23,7 +23,6 @@
 		"jest": "^20.0.4",
 		"lerna": "2.0.0-rc.4",
```

Comment:
You don't need lerna here, it can be safely deleted

Reply:
sure 👍